src/patternfly/components/Card/card.scss

```
@@ -331,7 +329,7 @@
   align-self: flex-start;
   order: 1;
   padding-left: var(--pf-c-card__actions--PaddingLeft);
-  margin: var(--pf-c-card__header-toggle--MarginTop) var(--pf-c-card__header-toggle--MarginRight) var(--pf-c-card__header-toggle--MarginBottom) auto;
+  margin: var(--pf-c-card__actions--MarginTop) auto var(--pf-c-card__actions--MarginBottom) auto;
```

Should this be `0` instead of auto?

```suggestion
  margin: var(--pf-c-card__actions--MarginTop) 0 var(--pf-c-card__actions--MarginBottom) auto;
```